__tests__/utils.test.js

```diff
@@ -102,3 +107,17 @@ describe("formatComments", () => {
     expect(formattedComments[0].created_at).toEqual(new Date(timestamp));
   });
 });
+
+describe("checkIfArticleExists",()=>{
+  test("should return true if the article exists",() => {
+    return checkIfArticleExists(1).then((result)=>{
+      expect(result).toBe(true)
+    })
+  })
+
+  test("should return false if the article does not exist",() => {
+    return checkIfArticleExists(20).then((result)=>{
+      expect(result).toBe(false)
+    })
+  })
+})
```

Comment:
You've added an afterAll, which is great, but skipped a beforeEach. Right now this works, because you're not doing any destructive queries, but once you start doing them in your other file, then these tests might start to break.